tests/fuzzer/fuzzer.py

```diff
@@ -40,7 +41,11 @@
         command += " ".join(chosen_words) + "\n"
 
     sock = socket.socket()
-    sock.connect(("localhost", 6667))  # If this fails you forgot to start mantatail
+    try:
+        sock.connect(("localhost", 6667))
+    except:
```

Comment:
Be specific about what error you're catching. If it's a KeyboardInterrupt or a bad file descriptor error, we'd rather just show the full error to user.

```suggestion
    except ConnectionRefusedError:
```